content/api/trafiklab-apis/gtfs-sweden/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: GTFS Sweden
+layout: "single"
+weight: 10
+aliases:
+  - /api/gtfs-sweden-static-data
+  - /api/gtfs-sweden-realtime
+---
+
+GTFS Sweden is a high quality GTFS feed containing most of the public transport data of Sweden. 
+
+### What does this dataset contain?
+
+This dataset contains high quality detailed data, both static and real-time, in the GTFS format.  
+This dataset is an aggregated dataset of all the different datasets in the [GTFS Regional API](/api/trafiklab-apis/gtfs-regional/).
+There is a single GTFS feed for static data. The realtime feeds are split up by specific regions or operators (for performance reasons). 
```

Comment:
The realtime feeds are, for performance reasons, split up by specific regions or operators.